server/package.json

```diff
@@ -7,12 +7,14 @@
     "build": "npm run build-config",
     "prebuild-config": "rm -fv dist/config*.js",
     "build-config": "NODE_ENV=production node build-config.js",
+    "gen-robot": "node gen-robot-txt.js",
+    "gen-sitemap": "node gen-sitemap-xml.js",
     "gen-lambda": "node cloudfront-gen-lambda.js",
     "security-checks": "npm audit --production --audit-level=high --registry=https://registry.npmjs.org/"
   },
   "engines": {
-    "node": ">=10.x",
-    "npm": ">=6.x"
+    "node": ">=12.x",
+    "npm": ">=7.x"
```

Comment:
is it necessary to make this upgrade here ?

Reply:
Not really, I can create a specific PR for this ?

Reply:
yep please, that could make more sense to isolate it